src/app/frontend/logs/stateconfig.js

```diff
@@ -27,16 +27,10 @@ import {stateName} from './state';
  */
 export default function stateConfig($stateProvider) {
   $stateProvider.state(stateName, {
-    url: `${appendDetailParamsToUrl('/log')}/:container`,
-    params: {
-      'container': {
```

Comment:
hmmm...container is optional, thus (as far as I understand) default value is null and squash is true. However, resourceType is not optional.